stdlib/public/core/CharacterProperties.swift

```diff
@@ -48,11 +56,65 @@ extension Character {
   ///     // lf.asciiValue == 10
   ///     let crlf = "\r\n" as Character
   ///     // crlf.asciiValue == 10
-  @inlinable
+  @_alwaysEmitIntoClient @_silgen_name("inlineASCIIValue")
```

Comment:
Same note on property descriptor and symbol naming.

```suggestion
  @_alwaysEmitIntoClient @_silgen_name("_inlineASCIIValue")
```